src/preparation.py

```diff
@@ -6,21 +6,20 @@
 import scipy.ndimage as ndi
 import torch
 from PIL import Image
-from imageio import imwrite
+from cv2 import imwrite
 from skimage.transform import resize
+from skimage.morphology import watershed, dilation, rectangle
 from sklearn.cluster import KMeans
 from torchvision import models
 from tqdm import tqdm
+from sklearn.model_selection import train_test_split
 
 
-def train_valid_split(meta, validation_size, valid_category_ids=None):
+def train_valid_split(meta, validation_size):
     meta_train = meta[meta['is_train'] == 1]
-    meta_train_split, meta_valid_split = split_on_column(meta_train,
-                                                         column='vgg_features_clusters',
-                                                         test_size=validation_size,
-                                                         random_state=1234,
-                                                         valid_category_ids=valid_category_ids
-                                                         )
+
+    meta_train_split, meta_valid_split = train_test_split(meta_train, test_size=validation_size, random_state=1234)
```

Comment:
@taraspiotr I think we should have `random_state` parametrized in pipeline_config.py, as it was in Mapping Challenge.

Reply:
@apyskir agree 100%